### PR TITLE
Update to db sync 13

### PR DESCRIPTION
**Pull request by pedromtcosta**

### CODE REVIEW

**Review by pedromtcosta — COMMENTED** [review]
/check

**Review by pedromtcosta — COMMENTED** [review]
/check

File: src/utils/index.ts [hunk 1]
```diff
@@ -52,7 +52,7 @@ export const applyRoutes = (routes: Route[], router: Router) => {
   for (const route of routes) {
     const { method, path, handler } = route;
     // uncomment this line if you want to test locally
-    // (router as any)[method](`/api${path}`, handler);
+    (router as any)[method](`/api${path}`, handler);
```
Comment (SebastienGllmt) [inline src/utils/index.ts:55]: Probably you didn't mean to commit this

Comment (pedromtcosta) [reply src/utils/index.ts]: Thank you Sebastien

**Review by vsubhuman — APPROVED** [review]

**Pull request merged**